assets/scss/lib/partials/_search.scss

```diff
@@ -56,3 +56,56 @@
     display: none;
   }
 }
+
+// Search page form.
+#views-exposed-form-localgov-sitewide-search-sitewide-search-page {
```

Comment:
Is there a class we can use instead of an id for theming? Following Drupal standards we should not theme by ID.

Reply:
I have updated this to use classes provided by views instead of the div